packages/components/src/utils/colors-values.js

```diff
@@ -101,15 +134,24 @@ export const ALERT = {
 	green: '#4ab866',
 };
 
+// Namespaced values for raw colors hex codes
+export const UI = {
+	background: BASE.white,
+	border: BASE.black,
+	borderFocus: BLUE.medium.focus,
+};
+
 export const COLORS = {
 	...BASE,
-	darkGray: DARK_GRAY,
+	darkGray: merge( DARK_GRAY, G2.darkGray ),
```

Comment:
This is mutating `DARK_GRAY`, such that every reference to `DARK_GRAY` would subsequently contain properties of `G2.darkGray`. I don't think that's the intention?

```suggestion
	darkGray: merge( {}, DARK_GRAY, G2.darkGray ),
```

A few other instances in this file as well.

Reply:
Oh! I didn't know it mutates. Thank you! Will fix those up